test/audits/geo-brand-presence.test.js

```diff
@@ -56,11 +56,11 @@ describe('Geo Brand Presence Handler', () => {
     };
     s3Client = {
       send: sinon.stub()
-          .callsFake((cmd) => {
-            const { name } = cmd.constructor;
-            const input = JSON.stringify(cmd.input, null, 2).replace(/\n[ ]*/g, ' ');
-            throw new Error(`no stubbed response for ${name} ${input}`)
-          }),
+        .callsFake((cmd) => {
+          const { name } = cmd.constructor;
+          const input = JSON.stringify(cmd.input, null, 2).replace(/\n[ ]*/g, ' ');
+          throw new Error(`no stubbed response for ${name} ${input}`)
+        }),
```

Comment:
hey, if you don't like the formatting, that's ok.
Please submit separate PRs in the future.
Unrelated changes tend to confuse reviewers :) 
Ok to leave it in this time.